nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -210,6 +210,19 @@ Costmap2DROS::on_configure(const rclcpp_lifecycle::State & /*state*/)
     layered_costmap_->getCostmap(), global_frame_,
     "costmap", always_send_full_costmap_);
 
+  auto layers = layered_costmap_->getPlugins();
+
+  for (auto & layer : *layers) {
+    auto costmap_layer = std::dynamic_pointer_cast<CostmapLayer>(layer);
+    if (costmap_layer != nullptr) {
+      layer_publishers_.emplace_back(
+          std::make_unique<Costmap2DPublisher>(
+              shared_from_this(),
+              costmap_layer.get(), global_frame_,
+              layer->getName(), always_send_full_costmap_));
```

Comment:
Is this name already namespaced by the costmap namespace (e.g. local/global)? That way there are not conflicting publications to the same topic by multiple costmaps that share a `static_layer` entry, for instance?